src/libbpf_internal.h

```diff
@@ -83,11 +83,8 @@ do {				\
 #endif
 /*
  * Re-implement glibc's reallocarray() for libbpf internal-only use.
- * reallocarray(), unfortunately, is not available in all versions of glibc,
- * so requires extra feature detection and using reallocarray() stub from
- * <tools/libc_compat.h> and COMPAT_NEED_REALLOCARRAY. All this complicates
- * build of libbpf unnecessarily and is just a maintenance burden. Instead,
- * it's trivial to implement libbpf-specific internal version and use it
```

Comment:
please don't change this file, it gets auto-synced

Reply:
Ah, I wasn't watching out for that.  Amended.